.bazelrc

```diff
@@ -15,8 +15,5 @@
 # The following flags are set to test use of new features for python toolchains
```

Comment:
lets just remove all this file, the -incompatible_use_python_toolchains should no longer be needed either

Reply:
can you also remove the bazelrc files in ./testing/examples/.bazelrc and ./testing/download_pkgs_at_root/.bazelrc